repository/SmalltalkCI-Core.package/SmalltalkCI.class/methodProperties.json

```
@@ -126,6 +126,7 @@
 		"isCoverageTestingEnabled" : "PeterUhnak 10/20/2017 12:33",
 		"load" : "fn 11/20/2017 11:56",
 		"performMetacelloLoad:" : "fn 4/15/2016 10:47",
+		"preLoad:" : "Hogoww 12/13/2022 19:29",
```

This is unfortunately still incorrect. `preLoad` does not take an argument, so it should read:

```suggestion
		"preLoad" : "Hogoww 12/13/2022 19:29",
```